Meadowlark-js/services/meadowlark-fastify/src/Service.ts

```diff
@@ -125,10 +126,11 @@ export function buildService(): FastifyInstance {
     fastify.get(`/${stage}/loadDescriptors`, loadDescriptors);
 
     // Authorization server handlers
-    fastify.get(`/${stage}/oauth/client/:clientId`, getClientByIdHandler);
-    fastify.post(`/${stage}/oauth/client/:clientId/reset`, resetAuthorizationClientSecretHandler);
-    fastify.post(`/${stage}/oauth/client`, createAuthorizationClientHandler);
-    fastify.put(`/${stage}/oauth/client/*`, updateAuthorizationClientHandler);
```

Comment:
We made a mistake before: it is better to pluralize the route. I suppose that is arguable: however, the Ed-Fi API handles it that way, so this is at least consistent.